conf/pipeline/sarek/uppmax.config

```diff
@@ -0,0 +1,17 @@
+// sarek/uppmax specific profile config
+
+params {
+  config_profile_contact = 'Maxime Garcia (@MaxUlysse)'
+  config_profile_description = 'nf-core/sarek uppmax profile provided by nf-core/configs'
+}
+
+if (hostname ==~ "i.*") {
+  singularity {
+      cacheDir = params.containers_path
+  }
+}
+
+params {
+  igenomeIgnore = true
+  genomes_base = params.genome == 'GRCh37' ? '/sw/data/uppnex/ToolBox/ReferenceAssemblies/hg38make/bundle/2.8/b37' : '/sw/data/uppnex/ToolBox/hg38bundle'
```

Comment:
Don't get why we have two different locations but fair enough.

Reply:
That's only while iGenomes is not updated